src/components/common/SetAllowance/index.tsx

```diff
@@ -1,56 +1,54 @@
 import React from 'react'
 import styled from 'styled-components'
 
-import { Token } from '../../../util/types'
+import { Button } from '../../buttons/Button'
+import { ButtonType } from '../../buttons/buttonStylingTypes'
 
-const Wrapper = styled.div``
-
-const Title = styled.h2`
-  font-size: 16px;
-  letter-spacing: 0.4px;
-  line-height: 1.2;
-  margin: 0 0 20px;
-`
-
-const DescriptionWrapper = styled.div`
+const Wrapper = styled.div`
   align-items: center;
   border-radius: 4px;
+  border: solid 1px ${(props) => props.theme.colors.primary};
   display: flex;
-  padding: 21px 25px;
+  margin-bottom: 25px;
+  padding: 13px 16px;
 `
 
 const Description = styled.p`
-  font-size: 14px;
-  letter-spacing: 0.2px;
-  line-height: 1.4;
-  margin: 0 25px 0 0;
+  font-size: 15px;
+  line-height: 1.2;
+  margin: 0 10px 0 0;
+  color: ${(props) => props.theme.colors.primary};
+`
+
+const UnlockButton = styled(Button)`
+  font-size: 18px;
+  height: 32px;
 `
 
 interface Props {
-  collateral: Token
-  onUnlock: () => void
-  loading: boolean
+  collateral: any
```

Comment:
Add Token type to the collateral props

Reply:
Yeah... the data we are getting from `collateralToken` is not compatible with the `Token` type, so I had to do that.

It seems something on `src/types/generatedGQL.ts` has to be fixed. If I add the type, I get the following error: `Type 'GetPosition_position_collateralToken' is missing the following properties from type 'Token': symbol, address, decimals`